### PR TITLE
Link open in new tab and spelling mistake

### PR DESCRIPTION
**Pull request by samagragupta**
Issues : #61 and #60

### CODE REVIEW

File: src/components/About.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ const About = () => (
 			<div className="about__info">
 				<h1 className="heading--primary">About Us</h1>
 				<p className="paragraph">
-					<span className="paragraph__highlight">Indian Institute of Information Technology, Vadodara</span> (IIIT-V)
-					is one of the newly formed IIIT by the Ministry of Human Resource Development (
+					<span className="paragraph__highlight">Indian Institution of Information Technology, Vadodara</span> (IIIT-V)
```
Comment (aashutoshrathi) [inline src/components/About.js]: Please update your branch. 😖 
Why? Institute -> Institution?

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
😕

File: src/components/Footer.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Footer extends Component {
 					<div className="links">
 						<h2 className="footer-header"><u className="footer-border">Links</u></h2>
 						<div className="footer-item"><a className="footer-link" href="/">Home</a></div>
-						<div className="footer-item"><a className="footer-link" href="http://iiitv.ac.in">Institute Site</a></div>
+						<div className="footer-item"><a className="footer-link" target="_blank" href="http://iiitv.ac.in">Institute Site</a></div>
```
Comment (aashutoshrathi) [inline src/components/Footer.js]: Make it `target="_blank" rel="noopener noreferrer"`

Comment (samagragupta) [reply src/components/Footer.js]: Done

**Review by aashutoshrathi — APPROVED** [review]
✨

**Pull request merged**